app/dashboard/migrations/0053_auto_20190920_2018.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 2.2.4 on 2019-09-20 20:18
```

Comment:
this one is conflicting with master, but maybe just recreate it just before merged to avoid having to recreate it many times.